tests/e2e/core-tests/specs/merchant/wp-admin-order-searching.test.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests, */
+
+/**
+ * Internal dependencies
+ */
+const {
+	merchant,
+	clearAndFillInput,
+	selectOptionInSelect2,
+	searchForOrder,
+	createSimpleProduct,
+	addProductToOrder,
+} = require( '@woocommerce/e2e-utils' );
+
+let orderId;
+
+const runOrderSearchingTest = () => {
+	describe('WooCommerce Orders > Search orders', () => {
+		beforeAll(async () => {
+			await merchant.login();
+
+			await createSimpleProduct('Wanted Product');
+
+			// Create new order for testing
+			await merchant.openNewOrder();
+			await page.select('#order_status', 'Pending payment');
+			await page.click('#customer_user');
+			await selectOptionInSelect2('Customer', 'input.select2-search__field');
+
+			// Change the shipping data
+			await page.waitFor(1000); // to avoid flakiness
+			await clearAndFillInput('#_shipping_first_name', 'Tim');
```

Comment:
It might be worth considering pulling these values from the config, e.g. `const customerFirstName = config.get( 'addresses.customer.billing.email' );`. Let me know what you think.

Reply:
Alternatively we can use some dummy data like we have in the `default.json` file so this doesn't look so much like a real person's address and is instead mock data.

Reply:
I understand but wanted to keep less amount of code. If need I can update this later. These values are used only in this test.